libs/viewer/src/SimpleViewer.cpp

```diff
@@ -1244,22 +1247,22 @@ void SimpleViewer::updateUserInterface() {
 
                     setTextureIfPresent(tweaks.mBaseColor.isFile, tweaks.mBaseColor.filename, "baseColor");
 
-                    matInstance->setParameter("normalScale", tweaks.mNormalIntensity.value);
+                    matInstance->setParameter("tintColor", tweaks.mTintColor.value);
```

Comment:
note: this will treat `tweaks.mTintColor.value` as linear gamma color. I believe you are aware of it.

Reply:
I'm treating it consistently with the other color-esque quantities such as absorption or sheen color, although I'm not loving it but I'm also too chicken to change it right now.